### DISCUSSION TITLE
Update README Docs

### DISCUSSION THREAD

**Discussion by tlarnold10** [serilog-contrib/serilog-sinks-grafana-loki#192] [Ideas]
Please add additional details in README documentation. For example, what are some of the other methods that can be added to a new logger? 
This example could be greatly expanded upon:
```csharp
ILogger logger = new LoggerConfiguration()
    .WriteTo.GrafanaLoki(
        "http://localhost:3100")
    .CreateLogger();

logger.Information("The god of the day is {@God}", odin)
```
The code from the sample directory was helpful, but would like to see more examples in the README file. For example, adding a label or property.

**Comment by mishamyte** [comment]
Hi @tlarnold10!

I think for the current moment all available properties are described [in the project wiki](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/wiki/Application-settings) with explanation and CLR + JSON type.

Also it has one [page ](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/wiki/Enterprise-application-configuration-example) with an example of one enterprise project.

So, TBH, I don't see the reason for having info in multiple places, cause it will bring us more problem like consistency and etc and I think wiki is the best place for doc. And samples could be useful too, as you mentioned too

**Reply by tlarnold10** [reply]
Ultimately, your call. I just found it difficult to include this library in my app because I wasn't aware of all the options available and how to use them from the existing documentation.
